kcidb/io/schema/v3.py

```diff
@@ -26,8 +26,7 @@
 
 # A regular expression pattern matching strings containing revision IDs
 REVISION_ID_PATTERN = \
-    f"{GIT_REPOSITORY_URL_PATTERN}" \
-    f"@{GIT_REPOSITORY_COMMIT_HASH_PATTERN}" \
+    f"{GIT_REPOSITORY_COMMIT_HASH_PATTERN}" \
```

Comment:
It's not new in this PR, but maybe `GIT_COMMIT_HASH_PATTERN` would be better, i.e. without `REPOSITORY` for the same reason as with the `git_commit_hash` field.

Reply:
Yes, makes sense, I could do that as part of the fix for #85.